mods/ts/maps/float/map.yaml

```diff
@@ -0,0 +1,664 @@
+MapFormat: 11
+
+RequiresMod: ts
+
+Title: They All Float
+
+Author: Westwood Studios
+
+Tileset: TEMPERATE
+
+MapSize: 90,100
+
+Bounds: 2,4,86,86
+
+Visibility: Lobby
+
+Categories: Conquest, Original FS Map, Playable
+
+Players:
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: Random
+	PlayerReference@Creeps:
+		Name: Creeps
+		NonCombatant: True
+		Faction: Random
+		Enemies: Multi0, Multi1
+	PlayerReference@Multi0:
+		Name: Multi0
+		Playable: True
+		Faction: Random
+		Enemies: Creeps
+	PlayerReference@Multi1:
+		Name: Multi1
+		Playable: True
+		Faction: Random
+		Enemies: Creeps
+
+Actors:
+	Actor0: jfish
+		Location: 39,5
+		Owner: Neutral
```

Comment:
Not sure if this is intentional as it will make them not attack the player. I would suggest `Owner: Creeps` instead.

Reply:
Ty, changed to Creeps. Didn't know if there are any neutral Creeps.